### DISCUSSION TITLE
[Enhancement] [select-v2] el-select-v2希望能增加append-to属性，virtualized select component requires the “append-to” attribute

### DISCUSSION THREAD

**Discussion by grzhan** [element-plus/element-plus#18793] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select-v2`

## Description

It's useful for select component has the "append-to" attribute. For long list, a virtualized select component is more suitable, but it has no "append-to" attribute.

### Additional comments

_(empty)_

**Comment by grzhan** [comment]
希望el-select-v2 也添加append-to 属性

**Reply by shizhihuaxu** [reply]
下一个版本就有了 [这个 pr](https://github.com/element-plus/element-plus/pull/18798)
